ui/components/app/flask/snap-authorship/snap-authorship.js

```diff
@@ -43,48 +55,124 @@ const SnapAuthorship = ({ snapId, className }) => {
 
   const friendlyName = snapId && getSnapName(snapId, subjectMetadata);
 
+  // Expanded data
+  const versionHistory = snap?.versionHistory ?? [];
```

Comment:
If version history can be empty or undefined _for some reason_, then `installInfo` will be undefined, then usage of the `installInfo` should be handled for such case, otherwise it will crash the page.

Reply:
Previous two versions of the `ViewSnap` were handling this somehow conditionally. The current version I implemented was inheriting that logic _just in case_. It looks like backwards compatibility.